onadata/libs/tests/utils/test_viewer_tools.py

```diff
@@ -183,52 +179,3 @@ def test_create_attachments_zipfile_file_too_big(self, rpt_mock):
 
         self.assertTrue(rpt_mock.called)
         rpt_mock.assert_called_with(message[0], message[1])
-
-    @override_settings(TESTING_MODE=False, ENKETO_URL='https://enketo.ona.io')
-    @requests_mock.Mocker()
-    def test_get_enketo_single_submit_url(self, mocked):
-        """Test get_single_submit_url.
-
-        Ensures single submit url is being received.
-        """
-        request = RequestFactory().get('/')
-
-        mocked_response = {
-            "single_url": "https://enketo.ona.io/single/::XZqoZ94y",
-            "code": 200
-        }
-
-        enketo_url = settings.ENKETO_URL + "/api/v2/survey/single/once"
-        username = "bob"
-        server_url = get_form_url(
-            request, username, settings.ENKETO_PROTOCOL, True, xform_pk=1)
-
-        url = '{}?server_url={}&form_id={}'.format(
-            enketo_url, server_url, "tag_team")
-        mocked.get(url, json=mocked_response)
-        response = get_enketo_single_submit_url(
-            request, username, id_string="tag_team", xform_pk=1)
-
-        self.assertEqual(
-            response, 'https://enketo.ona.io/single/::XZqoZ94y')
-
-    @override_settings(TESTING_MODE=False, ENKETO_URL='https://enketo.ona.io')
-    @requests_mock.Mocker()
-    def test_get_single_submit_url_error_action(self, mocked):
-        """Test get_single_submit_url to raises EnketoError."""
-        request = RequestFactory().get('/')
-
-        enketo_url = settings.ENKETO_URL + "/api/v2/survey/single/once"
-        username = "Milly"
-        server_url = get_form_url(
-            request, username, settings.ENKETO_PROTOCOL, True, xform_pk=1)
-
-        url = '{}?server_url={}&form_id={}'.format(
-            enketo_url, server_url, "tag_team")
-        mocked.get(url, status_code=401)
-        msg = "There was a problem with your submissionor form."\
-            " Please contact support."
-        self.assertRaisesMessage(
-            EnketoError,
-            msg, get_enketo_single_submit_url,
-            request, username, "tag_team", 1)
```

Comment:
I wonder if we should recreate these tests...? _Seems that the `enketo_url` function now does what these functions used to do ? Haven't seen a test that ensures that function raises the correct error... might have missed it..._

Reply:
Hey, I removed those in favour of other already existing tests.
Included that in the commit message `Remove `get_enketo_single_submit_urls` in favour of `test_get_single_submit_url` in the `test_xform_viewset.py file`